pkg/minio/api-handlers.go

```diff
@@ -448,7 +450,7 @@ func (h objectAPIHandlersWrapper) PutBucketHandler(w http.ResponseWriter, r *htt
 func (h objectAPIHandlersWrapper) HeadBucketHandler(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 	defer mon.Task()(&ctx)(nil)
-	if err := h.bucketPrefixSubstitution(w, r); err != nil {
+	if err := h.bucketPrefixSubstitution(w, r, "HeadObject"); err != nil {
```

Comment:
Just want to make sure that 3rd argument shouldn't be "HeadBucket", should it?

Reply:
fixed